### DISCUSSION TITLE
More "Limit", Quantity of Lists on Stars List

### DISCUSSION THREAD

**Discussion by Pedr0Ross1** [community/community#20603] [Other Feature Feedback, Questions, & Ideas]
The title describes all. We want to create more lists to organize starts. Thanks. Happy Coding.

**Comment by radar-blip** [comment]
This has been asked several times. Meaning its been brought to their attention. If I remember right, I think I seen a staff member saying they were considering it. If I come across the exact thread again, ill paste link here.

Here: https://github.com/orgs/github-community/discussions/8633
Seems they are working on it.
Have a great day!

**Comment by radar-blip** [comment]
Updated from 16 to 32 Lists! 👍🏼

**Reply by Pedr0Ross1** [reply]
Hello, i already used 32 lists before open this discussion. 64 would be cool. Thanks.

**Reply by radar-blip** [reply]
lol they will have to update it again soon!

**Reply by Pedr0Ross1** [reply]
Very well friend. Thanks.
